LICENSE

```diff
@@ -2531,3 +2531,330 @@ LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
 ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+arch/sim/src/sim/up_checkstack.c
+=========================================
+
+Copyright (C) 2019 FishSemi. All rights reserved.
+Author: Fishsemi <fishsemi@fishsemi.com>
```

Comment:
we don't need FishSemi copyright, which is part of Xiaomi.

Reply:
@xiaoxiang781216 I do not have the Xiaomi SGA to check if it inludes those files. Can you check it ?

Reply:
FishSemi is derived from Pinecone. All Pinecone copyright already change to Apache in this patch: https://github.com/apache/incubator-nuttx/pull/1629
So, I think it's safe to apply the same rule here.

Reply:
done